### PR TITLE
Adding new carousel/gallery to MS

### PR DESCRIPTION
**Pull request by aislingbrock**
#### What does this do?

Adds "new" carousel/gallery to Skeleton

### CODE REVIEW

File: app/site/resources/view/templates/blocks/javascript.html.twig [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 	'@Mothership:Site::resources/assets/js/jquery.touch-swipe.js'
-	'@Mothership:Site::resources/assets/js/jquery.gallery.js'
 	'@Mothership:Site::resources/assets/js/jquery.menu.js'
-	'@Mothership:Site::resources/assets/js/jquery.carousel.js'
 	'@Mothership:Site::resources/assets/js/jquery.offcanvas.js'
 	'@Mothership:Site::resources/assets/js/jquery.basket.js'
+	'@Mothership:Site::resources/assets/js/ms-carousel.min.js'
 	'@Mothership:Site::resources/assets/js/jquery.setup.js'
 
 	output='/assets/js/ms-site-*.js'
```
Comment (kuiche) [inline app/site/resources/view/templates/blocks/javascript.html.twig:10]: Can you reinstate the ?jsmin filter below. Probably my fault, but can you fix it as you are here?

Comment (aislingbrock) [inline app/site/resources/view/templates/blocks/javascript.html.twig:10]: dammit sammit.

Comment (kuiche) [inline app/site/resources/view/templates/blocks/javascript.html.twig:10]: man i wish github had a like button on comments

**Comment by kuiche** [comment]
:+1:

**Pull request merged**